lib/pbench/server/unpack_tarballs.py

```diff
@@ -0,0 +1,110 @@
+import glob
+from logging import Logger
+import os
+from pathlib import Path
+
+from pbench.server import PbenchServerConfig
+from pbench.server.filetree import FileTree
+from pbench.server.utils import get_tarball_md5
+
+
+class Results:
+    def __init__(
+        self,
+        ntotal=0,
+        nsuccess=0,
+    ):
+        self.ntotal = ntotal
+        self.nsuccess = nsuccess
```

Comment:
This could be a named tuple,

```
class Results(NamedTuple):
    ntotal: int
    nsuccess: int
```

(Or an `@dataclass`, which we should perhaps use more than we do. The main advantage of `NamedTuple` is to retrofit it into a situation where existing clients, particularly many existing clients, already rely on unpacking a tuple.)